src/django/api/models.py

```diff
@@ -1630,13 +1630,19 @@ def other_names(self):
         }
 
     def extended_fields(self, contributor_id=None):
-        active_items = self.facilitymatch_set \
+        active_matches = self.facilitymatch_set \
                            .filter(status__in=[FacilityMatch.AUTOMATIC,
                                                FacilityMatch.CONFIRMED,
                                                FacilityMatch.MERGED]) \
                            .filter(is_active=True) \
                            .values_list('facility_list_item')
 
+        active_sources = self.facilitylistitem_set \
+                             .filter(source__is_active=True) \
+                             .values_list('id')
```

Comment:
2 things which mean we shouldn't need to worry about that:
 - The intersection should be happening not in Django, but at the database level, which  (see https://www.postgresql.org/docs/current/queries-union.html & https://docs.djangoproject.com/en/4.1/ref/models/querysets/#intersection), so the actual data being intersected is SQL rows, not tuples
 - We have a list of tuples bc we're doing `values_list` without passing `flat=True` as a parameter, which will return a tuple with 1 column per field passed to `values_list` - so each queryset will always evaluate to a list of 1-tuples, since we only pass `id`.

Reply:
Ah. Thanks for explaining those details. 👍